app/views/ExposureHistory/NextSteps.tsx

```diff
@@ -1,24 +1,34 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import { TouchableOpacity, View, StyleSheet } from 'react-native';
 import { useNavigation } from '@react-navigation/native';
 
 import { Typography } from '../../components/Typography';
 import { NavigationBarWrapper } from '../../components/NavigationBarWrapper';
 import { Screens, useStatusBarEffect } from '../../navigation';
+import getHealthcareAuthorities from '../../store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction';
+import healthcareAuthorityOptionsSelector from '../../store/selectors/healthcareAuthorityOptionsSelector';
 
 import { Buttons, Spacing, Typography as TypographyStyles } from '../../styles';
 
 const NextSteps = (): JSX.Element => {
+  const dispatch = useDispatch();
```

Comment:
Using redux here seems a bit heavy to me, is there a reason we went with it particular?

It also strikes me as something that we want to keep separate from the user selecting HA flow as this data likely will come from the HA at the distribution level. so it might be a good idea to allow these data sources to diverge.  What do you think about introducing an env var `HA_NAME=ha_name` and provide this via react-native-config?  This seems lightest weight at this point and we can add a context or subscribe to the gobal state store in a future commit if it turns out we need it after learning more about this feature.